src/together/resources/finetune.py

```diff
@@ -91,7 +110,10 @@ def createFinetuneRequest(
         n_checkpoints=n_checkpoints,
         batch_size=batch_size,
         learning_rate=learning_rate,
+        lr_scheduler=lrScheduler,
         warmup_ratio=warmup_ratio,
+        max_grad_norm=max_grad_norm,
+        weight_decay=weight_decay,
```

Comment:
FinetuneRequest does not seem to have those fields, should we add them?

Reply:
Ty, updated